lib/ui/page/home/page/chat/widget/notes_block.dart

```diff
@@ -42,8 +43,17 @@ class NotesBlock extends StatelessWidget {
       title: info ? 'label_info'.l10n : 'label_chat_monolog'.l10n,
       titleStyle: style.fonts.large.regular.onBackground,
       children: [
+        Center(
+          child: Text(
+            'label_chat_monolog'.l10n,
+            style: style.fonts.small.regular.secondary,
+          ),
+        ),
```

Comment:
Isn't it a bit too much `Your notes` for a single page?

<img width="849" alt="Screenshot 2025-05-28 at 10 47 47" src="https://github.com/user-attachments/assets/e4ee4ae7-13ac-4211-b0dc-0be204c15e32" />